internal/error.go

```diff
@@ -119,14 +119,22 @@ Workflow consumers will get an instance of *WorkflowExecutionError. This error w
 */
 
 type (
+	ApplicationErrorOptions struct {
```

Comment:
This struct and fields should be documented.

Reply:
@Quinn-With-Two-Ns done